kotsadm/web/src/components/gitops/GitOpsDeploymentManager.jsx

```diff
@@ -192,13 +199,10 @@ class GitOpsDeploymentManager extends React.Component {
     const gitOpsInput = this.getGitOpsInput(provider, repoUri, branch, path, format, action, hostname);
 
     try {
-      const getGitOpsRepo = await this.getGitops();
-      if (getGitOpsRepo?.enabled) {
-        if (this.providerChanged()) {
-          const success = await this.resetGitOps();
-          if (!success) {
-            return false;
-          }
+      if (this.providerChanged()) {
```

Comment:
sorry, i didn't notice this before, but this should check for `this.state.gitops?.enabled` as well.